packageurl_test.go

```diff
@@ -1,572 +0,0 @@
-/*
```

Comment:
This file should be replaced with `packageurl_test.go` from https://github.com/package-url/packageurl-go/pull/80